_opcodes/op329-bgee.html

```diff
@@ -11,3 +11,9 @@
 pst: 0
 ---
 Applies the slow poison effect to the targeted creature(s), using the 'Slow Factor' field.
+
+{% capture note %}
+It only affects <a href="#op25">Poisons</a> if their param#2=(2 or 3), and only if they are not applied through an external EFF file.
```

Comment:
Are you sure it doesn't work also with mode 4?

Reply:
> Are you sure it doesn't work also with mode 4?

Yes, I'm sure. More info [here](https://forums.beamdog.com/discussion/comment/910650/#Comment_910650).

Reply:
I don't follow, type 4 can be applied normally too. Plus, since this is originally an iwd1 opcode, where there are also other poison types like snakebite.

Reply:
> I don't follow, type 4 can be applied normally too.

Are you sure about that? 'Type' 4 requires param#3 and param#3 can only be set in external EFF files...

Reply:
Ah, so the confusion is from which poison opcode is used, since it was different in iwd1. I guess they adapted it for iwdee to match the previous bgee behaviour.